Assets/Scripts/System/WorkspaceSaveSystem.cs

```diff
@@ -2,31 +2,41 @@
 using DynamicPanels;
 using System;
 using System.IO;
+using Zenject;
+using static Assets.Scripts.System.SettingsSystem;
```

Comment:
`using System.IO;` and `using static Assets.Scripts.System.SettingsSystem;` are not needed